classes/Kohana/Jam/Validator/Rule/Present.php

```diff
@@ -32,6 +33,14 @@ public static function is_empty_value($value, $allow_zero = FALSE)
 		return $allow_zero ? ( ! is_numeric($value) AND ! $value) : ! $value;
 	}
 
+	public static function is_empty_price($value, $allow_zero = FALSE)
+	{
+		if ($value instanceof Jam_Price) {
```

Comment:
You could make this `if (class_exists('Jam_Price') AND $value instanceof Jam_Price)` so it works when monetary is not installed. At the moment it would work based on a PHP quirk that it doesn't check if classes exist in type checks.